oddbird/sass-site
# Highlight styles

![](https://source.unsplash.com/featured/?cute,animal&afadsf)


## Description
Edit highlight styles

- make highlight colors variables (especially if used more than once)
- edited enhanced a color or two in order to differentiate it from the other similar reds colors (mostly the magenta ones).
- added the 600 weight (and italics) so the browser doesn't force a weight/style that didn't exist (most noticeable in Safari)

## Steps to test/reproduce
- review code samples on the documentation pages (not JS pages) ie. This is a good page to see some of the changes https://deploy-preview-56--sass-site-oddbird.netlify.app/documentation/interpolation/#in-sass-script

## Show me

Note: The items that are highlighted in a completely different color is a result of the library we are using now which doesn't wrap quite as much in spans like the previous library did.
![after-left-before-right](https://github.com/oddbird/sass-site/assets/1581694/d2778a3d-1930-485f-b119-46b5ef17b86e)
